src/mixxx.cpp

```diff
@@ -138,7 +170,9 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     QString resourcePath = m_pConfig->getResourcePath();
     initializeTranslations(pApp);
 
-    initializeFonts();
+    initializeFonts(); // long
```

Comment:
does the comment means long as in "takes a long time" or as in "32-bits integer" ?